### ISSUE TITLE
Adding "spinner" och "loader" when searching

### ISSUE THREAD

**Issue by enkelmedia** [marcemarc/tooorangey.DictionaryItemKeySearch#5]
Since the search sometimes can take a while it would be good to add a spinner, loader or something that indicates that a search is going on. I might be open to help with a PR if you like the idea?

**Comment by marcemarc** [comment]
Yep, that's a great idea and will improve the UX when search is slow, it's a little down on my endless list of things, but you raising the issues bumps it up a bit (it always surprises me when people actually use a package I've created!) but for sure a pull request would be most welcome!

**Comment by enkelmedia** [comment]
Hehe =D I know the feeling =D No worries, I just wanted to check before I start to work on something. I'll keep you posted here.
